t/t0001-basic.t

```diff
@@ -409,6 +409,24 @@ test_expect_success 'no unit tests built with libtool wrapper' '
 	test_must_fail grep -q "\.libs" test_dirs
 '
 
+test_expect_success 'command --help works outside of flux instance' '
+        flux --help 2>&1 |  grep -i usage &&
+        flux-R --help 2>&1 |  grep -i usage &&
```

Comment:
Why are you running the `flux-CMD` version of the commands instead of `flux CMD`, as users would? Just curious.

Also, it might be easier to maintain this list going forward to have the commands to test in a list, then loop through them running `$CMD --help` each time. Then in the future if we have a way for `flux(1)` to dump a list of all available commands (e.g. for use with command completion), we can switch to that.

Reply:
oh haha, i associated "not in an instance" with "flux command won't search paths", but that's not true.  I'll fixup and add the python commands too.